### ISSUE TITLE
No Intro Text in Linux build

### ISSUE THREAD

**Issue by MrCraigen** [NPBruce/valkyrie#1188]
### Description of Problem or Question
There is no intro text in any of the missions that you can choose. 
It's just either black screen or the choosen background image from the author. 
The background music and sounds works just fine. 

I have only tried a few of the Mansions of Madness missions.

### Valkyrie Version
v2.4.00b and v2.3.04

**Comment by NPBruce** [comment]
What distribution/version of linux do you have? Can you please provide a screen shot showing the problem?

Thanks.

**Comment by MrCraigen** [comment]
Im using Manjaro Linux x86_64, i took pictures from the 3 most popular English missions.

A Strain on Reality
[https://imgur.com/a/Yc5X98a](url)

Stress and Strain
[https://imgur.com/a/LYnvRrP](url)

Chapeldown Manor
[https://imgur.com/a/xW0YKgv](url)
[https://imgur.com/a/gdYkUBu](url)

I hope these will help

**Comment by NPBruce** [comment]
Looks like it is crashing out. Are you able to get the log from here:

~/.config/unity3d/NA/Valkyrie/Player.log

Thanks.

**Comment by MrCraigen** [comment]
Yep, I made a new Player.log for each mission. Not sure if it will be any different.

A Strain on Reality
https://www.paste.org/99942

Stress and Strain
https://www.paste.org/99943

Chapeldown Manor
https://www.paste.org/99944

**Comment by NPBruce** [comment]
Will investigate further

**Comment by krotax01** [comment]
Same here on Ubuntu 19.10.

**Comment by MrCraigen** [comment]
As of v2.4.09 the intro shows up as it should on the Linux client. 
Tested it today on Linux Mint 19.3 (Ubuntu 18.04).

Not sure what could have fixed it, maybe the update to Unity 2018.4.21f1 LTS?

I keep the ticket open for the devs. If they maybe want to have Player.log file to see any different.

EDIT:
I did the some test again with older version of Valkyrie. 

valkyrie-linux-2.4.07 (pre Unity 2018.4.21f1 LTS) = Broken intro
valkyrie-linux-2.4.08 (first version with Unity 2018.4.21f1 LTS) = Broken intro
valkyrie-linux-2.4.09 (post Unity 2018.4.21f1 LTS) = Working intro

So i guess it wasn't the new Unity version that fixed it :) not sure what fixed it. Maybe the recommend dlls that was included with the 2.4.09 release.
